pom.xml

```diff
@@ -123,6 +126,18 @@
             <version>${assertj-core.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.seleniumhq.selenium</groupId>
+            <artifactId>selenium-java</artifactId>
+            <version>3.141.5</version>
```

Comment:
Please, move all of the versions to the properties section, as you did in 1st homework